content/drafts/questions/SCHEMA.md

```
@@ -0,0 +1,313 @@
+Optional tags (recommended when relevant):
+
+| Tag | Values | Description |
+|-----|--------|-------------|
+| `treatments` | Array from canonical list below | Specific treatment tags (e.g., `buprenorphine`, `naltrexone`) |
+| `diagnoses` | Array of kebab-case slugs | Specific diagnoses (e.g., `opioid-use-disorder`) |
+
+### QID Format
+
+The `qid` must be globally unique across the entire question bank:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)